contracts/kill_switch/IssuesRegistry.sol

```diff
@@ -0,0 +1,28 @@
+pragma solidity 0.4.24;
+
+import "../apps/AragonApp.sol";
+import "./IIssuesRegistry.sol";
+
+
+contract IssuesRegistry is IIssuesRegistry, AragonApp {
+    bytes32 constant public SET_ENTRY_SEVERITY_ROLE = keccak256("SET_ENTRY_SEVERITY_ROLE");
+
+    mapping (address => Severity) internal issuesSeverity;
+
+    function initialize() external onlyInit {
+        initialized();
+    }
+
+    function setSeverityFor(address entry, Severity severity) external authP(SET_ENTRY_SEVERITY_ROLE, arr(entry)) {
+        issuesSeverity[entry] = severity;
+        emit SeveritySet(entry, severity, msg.sender);
+    }
+
+    function isSeverityFor(address entry) public view isInitialized returns (bool) {
```

Comment:
We should come up with a better name for this, perhaps `hasSeverity()` or `isAwareOfSeverity()`